packages/terra-framework-docs/src/terra-dev-site/doc/table/About.1.doc.mdx

```diff
@@ -19,15 +19,17 @@ import TablePropsTable from "terra-table/lib/Table?dev-site-props-table";
 
 ## Overview
 <Notice variant="important" ariaLevel="2">
-Due to poor support in VoiceOver for spanned table headings, Table sections are not currently recommended for applications intended to be used on Apple devices.
+Due to poor support in Apple VoiceOver for spanned table headings, the Table component is not currently recommended for applications used on Apple devices.
```

Comment:
This changes the context of this statement.  The use of the sections functionality specifically is not recommended.  This change makes consumers believe that the Table component itself is not recommended.

Reply:
I agree with @cm9361 comment in that this update changes context. I don't think we can say, or want to say, that the component as a whole isn't recommended for Apple devices.

Reply:
Thanks for clarifying this! I misunderstood because I thought the note applied to the whole component (as is usually the case when notes are placed at the top of a page). I'll update the wording. The same note exists under the actual Section heading -- should we keep it in both places, or remove the one from the overview?